public/views/delft/templates/login.tpl

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Template headers and footers are added automatically, so you don't have to add them. You can remove everything apart from ``#container``.